extension.bundle.ts

```diff
@@ -18,3 +18,8 @@ export { activateInternal, deactivateInternal } from './src/extension';
 // The tests should import '../extension.bundle'. At design-time they live in tests/ and so will pick up this file (extension.bundle.ts).
 // At runtime the tests live in dist/tests and will therefore pick up the main webpack bundle at dist/extension.bundle.js.
 export { ext } from './src/extensionVariables';
+import * as constants from './src/constants';
+export { constants };
+export { AzureTreeDataProvider, DialogResponses, TestAzureAccount, TestUserInput } from 'vscode-azureextensionui';
```

Comment:
nit: I prefer `export * from 'vscode-azureextensionui';` for all of these.  Just makes it easier so you don't have to go back and individually add something every time. Plus this file isn't actually used in production so it has no effect there